composer.json

```diff
@@ -9,28 +9,28 @@
     "require": {
         "composer/installers": "^2.0",
         "cweagans/composer-patches": "^1.6.7",
-        "drupal/core": "^9.4",
-        "drupal/core-composer-scaffold": "^9.4",
+        "drupal/core": "^10.0",
```

Comment:
As of Drupal 10, access checking must be explicitly specified on content entity queries (https://www.drupal.org/node/3201242).

- [JobListingHideMissingSubscriber.php#L81-L85](https://github.com/City-of-Helsinki/drupal-helfi-rekry/blob/UHF-8391/public/modules/custom/helfi_rekry_content/src/EventSubscriber/JobListingHideMissingSubscriber.php#L81-L85)
- [helfi_rekry_content.install#L213-L215](https://github.com/City-of-Helsinki/drupal-helfi-rekry/blob/UHF-8391/public/modules/custom/helfi_rekry_content/helfi_rekry_content.install#L213-L215)